dist/components/dropdown.js

```diff
@@ -1451,6 +1451,7 @@ $.fn.dropdown = function(parameters) {
                     module.event.item.click.call($selectedItem, event);
                     if(module.is.searchSelection()) {
                       module.remove.searchTerm();
+                      $search.focus();
```

Comment:
Why is this line added but the src not provided?

Reply:
I am confused.. Which source is missing?

Reply:
This is the dist file

Reply:
:fearful: My bad...now it's there.  :smile: